irrep/spacegroup.py

```diff
@@ -864,7 +862,7 @@ def get_irreps_from_table(self, kpname, K):
         tab = {}
         for irr in table.irreps:
             if irr.kpname == kpname:
-                k1 = np.round(np.linalg.inv(self.refUC).dot(irr.k), 5) % 1
+                k1 = np.round(np.linalg.inv(self.refUC.T).dot(irr.k), 5) % 1
                 k2 = np.round(K, 5) % 1
```

Comment:
Last equation in section "Vectors in reciprocal space" in the documentation I sent you by email.